src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -1146,6 +1146,67 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/nodes/{nodeId}/request-content-url':
+    post:
+      x-alfresco-since: "7.0.0"
+      tags:
+        - nodes
+      summary: Generate a direct access content url
+      description: |
+        **Note:** this endpoint is available in Alfresco 7.0 and newer versions.
+
+        Generate a direct access content url for the given **nodeId**.  
+        Optionaly the expiry at date could be set, so the direct access link would become invalid when the expiry date is reached. For example:
+
+        ```JSON
+         {
+           "expiresAt": "2017-03-23T23:00:00.000+0000"
+         }
+        ```
+        Or optionaly the validFor (in seconds) could be set, so the direct access link is valid for that length of time.
```

Comment:
typo "optionaly" -> "optionally" again, I won't add anymore comments but there are others in the file.